### DISCUSSION TITLE
How to fix this "Warning: fetchStandardFontData: failed to fetch file "LiberationSans-Regular.ttf" with "UnknownErrorException"

### DISCUSSION THREAD

**Discussion by sujjeee** [mozilla/pdf.js#17641] [Q&A]
Hey, I really don't know how to fix this warning:

## Warning: fetchStandardFontData: failed to fetch file "LiberationSans-Regular.ttf" with "UnknownErrorException: The standard font "baseUrl" parameter must be specified. Ensure that the "standardFontDataUrl" API parameter is provided.

If any of you know how to fix this, please let me know.

**Comment by sujjeee** [comment]
fixed

**Reply by tstansQAP360** [reply]
Mind sharing how you fixed it? I'm having the same problem!

**Reply by johtso** [reply]
Also interested in how you stopped this!

**Reply by zeke-john** [reply]
how did u fix this @sujjeee

**Reply by sujjeee** [reply]
im not using library just done like this 

```
import { PDFDocumentProxy } from "pdfjs-dist/types/src/display/api"

pdfjsLib.GlobalWorkerOptions.workerSrc =
"https://cdn.jsdelivr.net/npm/pdfjs-dist@4.3.136/build/pdf.worker.mjs"

const pdfDoc: PDFDocumentProxy = await pdfjsLib.getDocument(pdfbuffer).promise
```
